src/python/grpcio/grpc/_channel.py

```diff
@@ -99,6 +99,22 @@ def _wait_once_until(condition, until):
     else:
       condition.wait(timeout=remaining)
 
+_INTERNAL_CALLERROR_MESSAGE = (
+    'Internal gRPC CallError %d. ' +
```

Comment:
And just plain old ordinary "call error" here, since what semantically matters in this user-visible (or developer-visible) string is not the specific `CallError` class name.